content/handbook/Interim_processes/decision_making_process.md

```diff
@@ -0,0 +1,56 @@
+
+CONSENSUS AGREEMENTS: a [*consensus agreement*](https://www.seedsforchange.org.uk/consensus#flowchart) is a way of reaching agreement by  by discussing all opinions, ideas and concerns and collaborating to find solutions that everyone involved actively support, or at least can live with. While there are many ways of forming consensus agreements - with different degrees of structure and formality as appropriate to the group. The key to making consensus work is for everyone to express their needs and viewpoints clearly, and for the group to use this information to find a solution which builds on the common ground and resolves differences. The stages of a consensus process typically follow some variation of the following stages: introduction; clarification; open discussion (focusing on expressing different perspectives); exploration (focusing on potential solutions); proposal; amendments; agreement-test; and implementation. It is important to remember that consensus agreements function best when everyone shares a common goal and are willing to work together towards it. Finding a solution usually relies on people being flexible about their preferences in order to meet all the cores needs. This requires a lot of trust. 
+
+CONSENT AGREEMENTS: a [*consent agreement*]() is similar to consensus however, instead of seeking the best decision for the group, consent is the absence of objections (“I can live with this decision”). Rather then aiming to resolve all concerns, the proposal is only amended for “principled objections” (such as “I think there is a serious risk this proposal could do harm”, not “I don’t like it”). While there are many ways of reaching consent, the stages of reaching consent agreements typically includes some version of the following steps:  proposal; questions: reactions; amendments; confirmation; implementation. This approach is not appropriate in all contexts but does provide a transparent way to make low-risk time-sensitive decisions within decentralised groups.
+
+## Consensus Agreement for Initial Major Decisions
+Initially, our major decisions will be about how we function as a collective, including the principles informing our collective practices and how we make different types of decisions together. 
+
+While there are many ways of forming consensus agreements, to start us off we are adopting a 4-stage version: 
+
+1. INTRODUCTION, OPEN DISCUSSION, & EXPLORATION:   
+A facilitator starts a thread on Loomio, introducing an upcoming decision-point for the collective and open a question round discussion to clarify any points, express different perspectives, and explore potential pathways forward. If an Assembly occurs during this time, the topic will be included on the agenda for some in-person sense-making.
```

Comment:
Maybe we should be a bit clearer on what "in-person sense-making" means. Is "sense-making" just "discussion"?